runtime/kusama/src/lib.rs

```diff
@@ -1533,6 +1562,23 @@ pub mod migrations {
 	);
 	pub type V0943 = (
 		SetStorageVersions,
+
+		// Gov v1 storage migrations
+		// https://github.com/paritytech/polkadot/issues/6749
+		pallet_elections_phragmen::migrations::unlock_and_unreserve_all_funds::UnlockAndUnreserveAllFunds<Runtime>,
+		pallet_democracy::migrations::unlock_and_unreserve_all_funds::UnlockAndUnreserveAllFunds<Runtime>,
+		pallet_tips::migrations::unreserve_deposits::UnreserveDeposits<Runtime, ()>,
+
+		// RemovePallets only after they have been removed from the runtime. Otherwise, the on-chain
+		// storage version is removed for active pallets causing try-runtime to fail. The below code
+		// should be uncommented when the pallets are removed from the runtime.
+		// frame_support::migrations::RemovePallet<DemocracyStr, RocksDbWeight>,
```

Comment:
Yes, next upgrade when we remove the pallets from the runtime. 

This will be either 1.1 or 1.2 (depending if this makes it into 1.0), I'll update the comment with more info.